test/test_connection.py

```diff
@@ -1,11 +1,17 @@
-import unittest
+import datetime
+import sys
+if sys.version_info > (2, 6):
+    import unittest
+else:
```

Comment:
I have the same question as @Lukasa here. unittest2 causes far more headaches when used like this than not (in my experience).